corehq/apps/ota/views.py

```diff
@@ -140,7 +153,8 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
 
     project = Domain.get_by_name(domain)
     app = get_app(domain, app_id) if app_id else None
-    async_restore = toggles.ASYNC_RESTORE.enabled(domain)
+    async_restore = (toggles.ASYNC_RESTORE.enabled(domain) and
+                     float(openrosa_version) >= OPENROSA_VERSION_MAP['ASYNC_RESTORE'])
```

Comment:
nit: might make more sense to do with [`distutils`](http://stackoverflow.com/a/6972866/8207)